assets/_scss/components/_accordions.scss

```diff
@@ -0,0 +1,46 @@
+#accordion-menu {
+  border-color: $color-grey-600;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom-width: 0;
+  color: $color-grey-900;
+  margin: 0;
+  padding: 0;
+
+  li {
```

Comment:
This is a good point in moving these to classes, but again I think that would be something we'd want to test for.

Also, these pieces will be part of code snippets in which users are able to change/modify these snippets as they see fit. It will be interested to see how folks use, change and interact with them!